README.md

```diff
@@ -40,6 +40,138 @@ Then require the CSS in your `application.css` file:
 
 ## Usage
 
+### Rails >= 5.1
+
+To get started, just use the `bootstrap_form_with` helper
+in place of `form_with`. Here's an example:
+
+```erb
+<%= bootstrap_form_with(model: @user, local: true) do |f| %>
+  <%= f.email_field :email %>
+  <%= f.password_field :password %>
+  <%= f.check_box :remember_me %>
+  <%= f.submit "Log In" %>
+<% end %>
+```
+
+This generates:
+
+```html
+<form role="form" action="/users" accept-charset="UTF-8" method="post">
+  <input name="utf8" type="hidden" value="&#x2713;" />
+  <div class="form-group">
+    <label class="required" for="user_email">Email</label>
+    <input class="form-control" type="email" value="steve@example.com" name="user[email]" />
+  </div>
+  <div class="form-group">
+    <label for="user_password">Password</label>
+    <input class="form-control" type="password" name="user[password]" />
+    <small class="form-text text-muted">A good password should be at least six characters long</small>
+  </div>
+  <div class="form-check">
+    <label class="form-check-label" for="user_remember_me">
+    <input name="user[remember_me]" type="hidden" value="0" />
+    <input class="form-check-input" type="checkbox" value="1" name="user[remember_me]" /> Remember me</label>
+  </div>
+  <input type="submit" name="commit" value="Log In" class="btn btn-secondary" data-disable-with="Log In" />
+</form>
+```
+
```

Comment:
I think it is sufficient to stop here, with a note that says that both `model:` and `url:` styles are supported. We don't need to reproduce all the Rails documentation. (Perhaps a link to the appropriate section of the Rails doc would be nice.) In particular, I would remove these sections as well:

* Important Differences Between `form_with` and `form_for`
* Ajax by Default
* No Default DOM IDs
* Use `fields` Instead Of `fields_for` In Nested Forms
* No Default Classes

Reply:
Back in September, when I was first trying my fork with `bootstrap_form_with`, this section brought together the things I was learning about `form_with` versus `form_for`. But now I'm sure I can find a good link to differences, so I'll take this section out and replace it with a link or two to other documentation.